### PR TITLE
Game Page Layout

### PR DESCRIPTION
**Pull request by Ishimstn**
I'll add my portion of the code into the main doc. Can one of you look over it and see if it looks okay. I'm having a bit of a problem trying to implement the avatar image onto the boxes.

### CODE REVIEW

**Comment by malenchite** [comment]
I'll take a detailed look when I get a chance. As for rendering the bot answers, take a look at renderBotAnswer() and other references to #botDiv in the JS

**Comment by malenchite** [comment]
I took a look over the design. Some comments:
* I suggest working with the other front-end members to make for a consistent experience between the other pages. Right now, this style is at odds with all the other work that has been done.
* Check out renderQuestion(), processAnswer(), #next-button onclick event, finishQuestion(), and endGame() for ideas as how the back-end of this was designed to work. These can be modified, of course, but you should make sure that IDs match and that the game behaves as expected.
* Why are you using input fields for the question and answers? These should be static text.
* The use of absolute positioning makes the page very non-responsive for mobile viewing

File: Trivia_Trainer.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <meta charset="UTF-8">
+    <head>
+        <title>Trivia_Trainer Gameplay Page</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="<link rel=stylesheet href="https://s3-us-west-2.amazonaws.com/colors-css/2.2.0/colors.min.css">"
```
Comment (malenchite) [inline Trivia_Trainer.html:10]: This link is formatted incorrectly as well as having a trailing quotation mark that shows up on the page.

File: Trivia_Trainer.html [hunk 2]
```diff
@@ -2,30 +2,40 @@
...
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
         <link rel="stylesheet" href= "Trivia_Trainer.css">
+        
     </head>
 
 
-    <body>
+<body>
+
+    <div class id = "Game-screen" class="Game-screen">
```
Comment (malenchite) [inline Trivia_Trainer.html:21]: There's an extra "class" in here that doesn't belong

File: Trivia_Trainer.html [hunk 3]
```diff
@@ -87,33 +97,40 @@ <h3>Players</h3>
...
             </div>
+        </div>
 
-            <div class = "row">
-                <div class = "col-md-3_1">
-                    <div class="col-md-9_1">
-                        <input type="text" class="form-control" placeholder = "Player 2 Answer Choice" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-default">
-                    </div>
+        <div class = "row">
+            <div class = "col-md-3_1">
```
Comment (malenchite) [inline Trivia_Trainer.html:113]: I'm not sure about these class names. They resemble boot-strap responsive columns, but with the _1 suffix they will not function responsively, which is confusing.

**Comment by Ishimstn** [comment]
I went ahead and sent in the revised copy of the game page

**Comment by malenchite** [comment]
I recommend checking in on the current state of the main branch. Work has already been largely completed, except for maybe some style polishing.

**Pull request closed without merging**